gnmvidispine/vidispine_api.py

```diff
@@ -352,9 +360,11 @@ def chunked_upload_request(self,upload_io,total_size,chunk_size,
         self.logger.debug("Commencing upload from {0} in chunks of {1}".format(upload_io,chunk_size))
         self.logger.debug("uploading to {0} with account {1}".format(self.host,self.user))
 
+        size_to_send = int(str(total_size))
```

Comment:
why do we need to convert to string then back to int? what is testfilesize?

Reply:
Because if you use int as is it prints an L on the end sometimes. The int conversion it to make sure it goes in to the call as an int. I know it looks silly, but this does work.

Reply:
testfilesize is the name of the object that was holding the number to be tested in the test code.

Reply:
total_size is an object holding the total size of the file. This is supplied from further up the code. I have checked this and the number is reliably right. 

If you are wondering why I am using the total size, it is because that is the correct syntax: -

http://apidoc.vidispine.com/latest/ref/item/import.html#import-using-the-request-body